infrastructure/merklemountainrange/benches/bench.rs

```diff
@@ -0,0 +1,169 @@
+#[macro_use]
+extern crate criterion;
+#[macro_use]
+extern crate serde_derive;
```

Comment:
serde derive is now included in the normal serde crate, just add the feature derive